doc/example-grids/cubedsphere_equidistant_fv3_sar.yml

```diff
@@ -0,0 +1,18 @@
+# Test for the regionally refined cubed-sphere grid that uses the equiangular 'FV3 flavour'
+# formulation and projection. In this mode a Schmidt scaling is used with a stretching factor to
+# effectively move grid points from one face to another. The target lon/lat is the center of the
+# grid points with shortest grid spacing.
+
+type   : "cubedsphere_equidistant_fv3"
+CubeNx : 12
+DoSchmidt: true
+StretchFac: 3
+TargetLon: -45.5
+TargetLat: 90.0
```

Comment:
I would have seen rather that this specific rotation/stretching be part of another projection, and be more consistent with the existing RotatedSchmidt projection:
```
projection:
    type: rotated_schmidt
    stretching_factor: 2.0
    north_pole: [-45.5, 90.]
```